File: downloadlib/src/main/java/com/king/mobile/downloadlib/TaskState.java
package com.king.mobile.downloadlib;

public class TaskState {
    public static final int STATE_URL_ERROR = -1;// 地址有误
    public static final int STATE_CREATED = 0;//初始状态
    public static final int STATE_WAITING = 1;// 任务分配完成，地址无误可以下载
    public static final int STATE_DOWNLOADING = 2;// 下载中
    public static final int STATE_PAUSE = 3;// 暂停中
    public static final int STATE_DOWNLOAD_ERROR = 4;// 下载出错
    public static final int STATE_FINISHED = 100;// 已完成
}

File: downloadlib/src/main/java/com/king/mobile/downloadlib/TaskState.kt
package com.king.mobile.downloadlib


class TaskState {
    companion object {
        val STATE_CREATE = 0  // 初始状态
        val STATE_ERROR = -1  // 地址有误
        val STATE_START = 1  // 地址无误可以下载
        val STATE_DOWNLOADING = 2 // 下载中
        val STATE_STOP = 3 // 暂停中
        val STATE_FINISHED = 4 // 已完成
    }
}